### DISCUSSION TITLE
How do I know if I'm creating or updating an item in a custom interface?

### DISCUSSION THREAD

**Discussion by mattiaz9** [directus/directus#5131] [Archive]
What I want to achieve is to provide a random hash when I'm creating a new item, but not when I'm updating an existing one.

Is there an 'elegant' way to check that, without looking if the url ends with a '+'?

**Comment by rijkvanzanten** [comment]
There's a `primaryKey` prop you can look for. If that's `+`, it's a new item 👍🏻
